oneanddone/functional_tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+bidpom
+pytest-selenium
+requests==2.8.1
+-e git+https://github.com/bobsilverberg/pytest-django.git@6d8d45c4fd1eb58b553045dd844e35885fcf67e6#egg=pytest_django
```

Comment:
No. If we could I wouldn't have done this. ;)

I needed to hack `pytest-django` in order to make my tests work. Without this hack Django was not serving static assets via the live_server and therefore the functional tests that need JavaScript (which include any that need to log in), were not working.

I did raise an issue against `pytest-django` for this, and I should document that in the requirements file, so I will.